### PR TITLE
Internet protocol

### PR DESCRIPTION
**Pull request by mrbloopbloop**
Added an overview of what packets, IP addresses, and ports are.
Included a list of common ports and a link to MIT's list of standard ports.

### CODE REVIEW

**Comment by nathandstevens** [comment]
Linking to issue #26

File: ip.html [hunk 1]
```diff
@@ -43,7 +43,7 @@ <h2>Packets</h2>
...
+	So next lets learn how to write the address that packets use. They are called IP addresses or IP's for short. An IP is made of four numbers ranging from 0 to 255. Each of them are separated by periods. Each of these numbers are held in 1 byte, and those four bytes make up the IP address. Each computer on the internet has at least one of these addresses. That includes servers for web pages. Another way you could access a webpage is by typing the IP address into the address bar of your browser. For example <code>142.250.138.100</code> is one of the IP's that leads to google.com. If you type that in, it should take you straight to google.
```
Comment (DGKSK8LIFE) [inline ip.html:46]: `let's`*

**Review by DGKSK8LIFE — CHANGES_REQUESTED** [review]

File: ip.html (hunk 1 above)
Comment (nathandstevens) [reply ip.html:46]: Good catch!

Comment (DGKSK8LIFE) [reply ip.html]: Thanks.

**Review by DGKSK8LIFE — APPROVED** [review]

**Pull request closed without merging**